applications/GeoMechanicsApplication/custom_strategies/schemes/geomechanics_time_integration_scheme.hpp

```diff
@@ -354,6 +350,17 @@ class GeoMechanicsTimeIntegrationScheme : public Scheme<TSparseSpace, TDenseSpac
         // intentionally empty
     }
 
+    virtual void CheckAllocatedVariables(const ModelPart& rModelPart) const
+    {
+        // intentionally empty
+    }
+
+    virtual void UpdateScalarTimeDerivative(Node& rNode,
+                                            const Variable<double>& variable,
+                                            const Variable<double>& dt_variable) const {
```

Comment:
A minor style issue:
```suggestion
                                            const Variable<double>& rVariable,
                                            const Variable<double>& rDtVariable) const {
```

Reply:
Done